<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>601</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1001</width>
    <height>601</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1001</width>
    <height>601</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Bahnschrift</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>SIME</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>981</width>
      <height>581</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 255);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="lblSime">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>390</y>
       <width>391</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 48pt &quot;Algerian&quot;;</string>
     </property>
     <property name="text">
      <string>S  I  M  E</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblIcon">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>391</width>
       <height>391</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 200pt &quot;Wingdings&quot;;
</string>
     </property>
     <property name="text">
      <string>&amp;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEstudiante">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>80</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(2, 188, 255);</string>
     </property>
     <property name="text">
      <string>ESTUDIANTE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDocente">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>80</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 85, 255);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>DOCENTE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMatricula">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>150</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(2, 188, 255);</string>
     </property>
     <property name="text">
      <string>MATRICULA</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSeccion">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>150</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 85, 255);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>SECCION</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCuota">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>220</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(2, 188, 255);</string>
     </property>
     <property name="text">
      <string>CUOTA</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnHorario">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>220</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 85, 255);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>HORARIO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConcepto">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>290</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(2, 188, 255);</string>
     </property>
     <property name="text">
      <string>CONCEPTO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCarrera">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>290</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 85, 255);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>CARRERA</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDetalle">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>360</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(2, 188, 255);</string>
     </property>
     <property name="text">
      <string>DETALLE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnModalidad">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>360</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 85, 255);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>MODALIDAD</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCurso">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>430</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(2, 188, 255);</string>
     </property>
     <property name="text">
      <string>CURSO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDiaSemana">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>430</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 85, 255);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>DIA SEMANA</string>
     </property>
    </widget>
    <zorder>lblIcon</zorder>
    <zorder>btnEstudiante</zorder>
    <zorder>btnDocente</zorder>
    <zorder>btnMatricula</zorder>
    <zorder>btnSeccion</zorder>
    <zorder>btnCuota</zorder>
    <zorder>btnHorario</zorder>
    <zorder>btnConcepto</zorder>
    <zorder>btnCarrera</zorder>
    <zorder>btnDetalle</zorder>
    <zorder>btnModalidad</zorder>
    <zorder>btnCurso</zorder>
    <zorder>btnDiaSemana</zorder>
    <zorder>lblSime</zorder>
   </widget>
  </widget>
  <action name="actionDocente">
   <property name="text">
    <string>Docentes</string>
   </property>
  </action>
  <action name="actionMatricula">
   <property name="text">
    <string>Matricular Estudiante</string>
   </property>
  </action>
  <action name="actionEstado_de_Pago">
   <property name="text">
    <string>Estado de Pago</string>
   </property>
  </action>
  <action name="actionHorario">
   <property name="text">
    <string>Horario</string>
   </property>
  </action>
  <action name="actionSeccion">
   <property name="text">
    <string>Seccion</string>
   </property>
  </action>
  <action name="actionConcepto">
   <property name="text">
    <string>Concepto</string>
   </property>
  </action>
  <action name="actionEstudiante">
   <property name="text">
    <string>Registrar Estudiante</string>
   </property>
  </action>
  <action name="actionDetalle">
   <property name="text">
    <string>Detalle</string>
   </property>
  </action>
  <action name="actionDia_Semana">
   <property name="text">
    <string>Dia Semana</string>
   </property>
  </action>
  <action name="actionModalidad">
   <property name="text">
    <string>Modalidad</string>
   </property>
  </action>
  <action name="actionCuota">
   <property name="text">
    <string>Cuota</string>
   </property>
  </action>
  <action name="actionCurso">
   <property name="text">
    <string>Curso</string>
   </property>
   <property name="toolTip">
    <string>Curso</string>
   </property>
  </action>
  <action name="actionCarrera">
   <property name="text">
    <string>Carrera</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
